app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -75,7 +75,7 @@ def set_date_header
         def set_cors_preflight_headers
           headers['Access-Control-Allow-Origin'] = '*'
           headers['Access-Control-Allow-Methods'] = 'GET, HEAD, POST, PUT, PATCH, DELETE, OPTIONS'
-          headers['Access-Control-Allow-Headers'] = 'X-Requested-With, X-Prototype-Version, X-CSRF-Token, Token'
+          headers['Access-Control-Allow-Headers'] = 'X-Requested-With, X-Prototype-Version, X-CSRF-Token, Token, Authorization'
           headers['Access-Control-Max-Age'] = '86400'
         end
```

Comment:
Two questions:

* Can we remove the wildcard Access-Control-Allow-Origin?  I'm thinking it should check the list of hosts like the tutor auth pr does at:  https://github.com/openstax/tutor-server/blob/cors-w-token/app/controllers/auth_controller.rb#L48-L55  The list of hosts could be set as a configuration in `config/initializers/openstax_api.rb`

* I'm not sure what the "Token" header does.  I tested setting the token to it but doorkeeper didn't grant access when using it.  Should we remove it?